README.md

```diff
@@ -242,15 +244,17 @@ When deploying for the first time, a Google Cloud Platform service account must
 1.  Run the command
 
     ```
-    cat $DSS_HOME/gcp-credentials.json | scripts/set_secret.py --secret-name $GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME
+    cat $DSS_HOME/gcp-credentials.json | ./scripts/dss-ops.py secrets set --secret-name $GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME
```

Comment:
actual question here: should we assume that the user is currently in the `$DSS_HOME` dir, so using `./scripts/` would resolve? 
It's something that i also do; but others sometimes say to omit.